AutomationHelpers/src/UserCodeCollections/StringLibrary.cs

```diff
@@ -24,5 +24,33 @@ public static string ConcatStrings(string value1, string value2)
         {
             return String.Concat(value1, value2);
         }
+
+        /// <summary>
+        /// Creates and returns a random string. If a string shorter than 35 characters is used it may not be unique.
+        /// </summary>
+        /// <param name="length">Expected length of the return value (default length is 35)</param>
+        /// <returns>Random string</returns>
+        [UserCodeMethod]
+        public static string GetRandomString(string length)
+        {
+            string returnValue = "";
+
+            if (length == null || length.Equals(""))
+            {
+                length = "35";
+            }
+
+            if (Int32.Parse(length) > 35)
+            {
+                returnValue = getRandomString((Int32.Parse(length) - 36).ToString());

```

Comment:
I don't think that this compiles, there is a difference in the capitalization of the method names

Reply:
fixed